File: C++/Estrutura repetitiva/impar.cpp
// Biblioteca(s)
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    // Variável
    int quant, i, num;

    // Entrada de dados
    cout << "Digite quantos numeros voce vai digitar: ";
    cin >> quant;

    // Estrutura repetitiva
    for (i=1; i <= quant; i++) {
        // Entrada de dados
        cout << "Digite um numero: ";
        cin >> num;

        // Estrutura condicional
        if (num < 0 && num % 2 != 0) {
            cout << "Impar negativo!" << endl;
        }
        else if (num < 0 && num %2 == 0) {
            cout << "Par negativo!" << endl;
        }
        else if (num == 0) {
            cout << "Nulo!" << endl;
        }
        else if (num > 0 && num % 2 != 0) {
            cout << "Impar positivo!" << endl;
        }
        else {
            cout << "Par positivo!" << endl;
        }
    }

    return 0;
}

File: C#/Estrutura repetitiva/impar.cs
// Variável
int quant, i, num;

// Entrada de dados
Console.Write("Digite quantos números você vai digitar: ");
quant = int.Parse(Console.ReadLine());

// Estrutura repetitiva
for (i = 1; i <= quant; i++)
{
    // Entrada de dados
    Console.Write("Digite um número: ");
    num = int.Parse(Console.ReadLine());

    /// Estrutura condicional
    if (num < 0 && num % 2 != 0)
    {
        Console.WriteLine("Ímpar negativo!");
    }
    else if (num < 0 && num %2 == 0)
    {
        Console.WriteLine("Par negativo!");
    }
    else if (num == 0)
    {
        Console.WriteLine("Nulo!");
    }
    else if (num > 0 && num %2 != 0)
    {
        Console.WriteLine("ímpar positivo!");
    }
    else
    {
        Console.WriteLine("Par positivo!");
    }
}